.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Bug Report or Feature Request (mark with an `x`)
+```
+- [ ] bug report -> please search issues before submitting
+- [ ] feature request
+```
+
+## Versions.
+```
+browser:
+browser version: 
+operating system: 
+```
+
+
+## Repro steps.
+<!--Simple steps to reproduce this bug.-->
+
+## Desired functionality.
```

Comment:
"Desired or expected functionality"

In case of a feature request, what would you like to see implemented? In case of a bug report, what did you expect to happen?